web/docs/setup-guides/deploy-a-node.md

```diff
@@ -63,6 +63,18 @@ dpkg -i tenzir-static-amd64-linux.deb
 
 [tenzir-debian-package]: https://github.com/tenzir/tenzir/releases/latest/download/tenzir-static-amd64-linux.deb
 
+</TabItem>
+<TabItem value="rpm_based" label="RPM based (RedHat / OpenSUSE / ...)">
+
+Download the latest [RPM package][tenzir-rpm-package] and install it via
+`rpm`:
```

Comment:
```suggestion
`dnf` or `yum`:
```

I think `dnf` is actually the more modern version.

Reply:
I think we should actually use `rpm -i` because that should work across all distros, whereas the high level commands `yum`, `dnf`, `zypper`,... are distro dependent.

Reply:
Even better.